sympy/functions/special/bessel.py

```diff
@@ -953,9 +953,13 @@ def jn_zeros(n, k, method="sympy", dps=15):
                                               int(l)), prec)
                 for l in range(1, k + 1)]
     elif method == "scipy":
-        from scipy.special import sph_jn
         from scipy.optimize import newton
-        f = lambda x: sph_jn(n, x)[0][-1]
+        try:
+            from scipy.special import spherical_jn
+            f = lambda x: spherical_jn(n, x)
+        except:
```

Comment:
Thanks! Done :)
However, my doubt is , why do we use only `ImportError` and not except in general? To be aware of any kind of new error that may arise in future?